content/blog/2023-10-05-october-Prism-launch/contents.lr

```diff
@@ -0,0 +1,22 @@
+
+Prism builds on a strong research foundation first made possible in the DigitalHub repository. “One of the most exciting features of Prism is the ability to create communities on topics, projects, or events” Gutzman says. Communities include open access research from Feinberg on [COVID-19](https://prism.northwestern.edu/communities/covid-19-community?q=&l=list&p=1&s=10&sort=newest), training presentations from the [Biostatistics Collaboration Center](https://prism.northwestern.edu/communities/biostatistics-collaboration-center-lecture-series?q=&l=list&p=1&s=10&sort=newest), and the [NUCATS Grant Repository](https://galter.northwestern.edu/exit?url=https%3A%2F%2Fredcap.nubic.northwestern.edu%2Fredcap%2Fsurveys%2F%3Fs%3DJKT3M84PM9LMPTKL), which is a centralized resource for grant writers and investigators provided by the NUCATS Institute. “Prism is an excellent home for the NUCATS Grants Repository, allowing us to easily share exemplar grant templates and other resources with investigators across the Northwestern University community,” says Dr. Richard D’Aquila, Director of NUCATS.
+
+Prism is built on the InvenioRDM software, which also forms a strong and sustainable foundation for Zenodo. “With its user-friendly interface and advanced features, InvenioRDM is truly a game-changer in the world of repositories. This platform is designed to make research more accessible and open to the public, promoting innovation and collaboration within the academic community,” says Holmes. Over the past several years, CERN and Northwestern have collaborated as core co-developers of the software, partnering with the global Invenio Open Source Community to develop InvenioRDM as a turnkey, scalable, and top-of-the-class user experience software for repositories. The InvenioRDM software offers a reliable environment for science, empowering preservation, credit, discovery, and sharing while maintaining integrity in its responsiveness to the evolving needs of the research community, including data sharing policy compliance. 
+
+
+Notably, Northwestern and CERN recently expanded this collaboration through an award from the [Generalist Repository Ecosystem Initiative (GREI)](https://galter.northwestern.edu/exit?url=https%3A%2F%2Fdatascience.nih.gov%2Fnews%2Fnih-office-of-data-science-strategy-announces-new-initiative-to-improve-data-access) from the NIH Office of Data Science Strategy (ODSS) to Zenodo to help researchers improve discoverability of their data and lead to greater reproducibility and reuse of data.
```

Comment:
It feels like an explicit final concluding paragraph/note would round things off nicely e.g., start that paragraph with "We look forward to the evolution of this platform. Notably, a recent award from [...] will help improve [...] in the coming years."

Reply:
Sounds good. I'm not aware of any recent awards, but if you and Sara have text to add - that would be great!

Reply:
Oh here the recent award is from GREI which will benefit our instance by benefiting the platform as a whole.

Reply:
Prism is built on the InvenioRDM software, which also forms a strong and sustainable foundation for Zenodo. “With its user-friendly interface and advanced features, InvenioRDM is truly a game-changer in the world of repositories. This platform is designed to make research more accessible and open to the public, promoting innovation and collaboration within the academic community,” says Holmes. Over the past several years, CERN and Northwestern have collaborated as core co-developers of the software, partnering with the global Invenio Open Source Community to develop InvenioRDM as a turnkey, scalable, and top-of-the-class user experience software for repositories. The InvenioRDM software offers a reliable environment for science, empowering preservation, credit, discovery, and sharing while maintaining integrity in its responsiveness to the evolving needs of the research community, including data sharing policy compliance. 

Notably, Northwestern and CERN recently expanded this collaboration through an [award](https://blog.zenodo.org/2022/09/30/2022-09-30-northwestern-cern-join-nih-in-enhancing-access-to-biomedical-research/) from the NIH Office of Data Science Strategy/Office of the NIH Director pursuant to OTA-21-009, [Generalist Repository Ecosystem Initiative (GREI)](https://datascience.nih.gov/data-ecosystem/generalist-repository-ecosystem-initiative), to Zenodo to help researchers improve discoverability of their data and lead to greater reproducibility and reuse of data.